lib/signs/bus.ex

```diff
@@ -407,6 +415,21 @@ defmodule Signs.Bus do
     {messages, audios}
   end
 
+  defp configs_content(nil, _), do: []
+
+  defp configs_content(configs, predictions_lookup) do
+    Enum.flat_map(configs, fn config ->
+      Stream.flat_map(config.sources, fn source ->
+        Map.get(predictions_lookup, {source.stop_id, source.route_id, source.direction_id}, [])
+      end)
+      |> Enum.group_by(&PaEss.Utilities.headsign_key(&1.headsign))
+      |> Enum.map(fn {_, list} ->
+        {:predictions, Enum.sort_by(list, & &1.departure_time, DateTime)}
+      end)
```

Comment:
Note: This is where the actual alert message generation will happen. If we don't find any predictions for a config, we can consult the alert data and produce an `:alert` structure here if appropriate. We'll then need to sort them to the end, and write code for formatting them and reading them.